rt_gene/scripts/estimate_blink_standalone.py

```diff
@@ -0,0 +1,116 @@
+#!/usr/bin/env python
+
+from __future__ import print_function, division, absolute_import
+
+from os import listdir
+from rt_bene.estimate_blink_base import BlinkEstimatorBase
+import sys
+import os
+import argparse
+
+script_path = os.path.dirname(os.path.realpath(__file__))
+
+class BlinkEstimatorStandalone(BlinkEstimatorBase):
+
+    def __init__(self, model_path, threshold, input_size, viz):
+        super(BlinkEstimatorStandalone, self).__init__("gpu", model_path, threshold, input_size)
+        self.viz = viz
+            
+
+class BlinkEstimatorImagePair(BlinkEstimatorStandalone):
+
+    def estimate(self, images):
+        right_eyes = [images[0]]
+        left_eyes = [images[1]]
+        probs, blinks = self.predict(right_eyes, left_eyes)
+
+        if self.viz:
+            #TODO: viz on eye images?
+            viz_img = self.overlay_prediction_over_img(right_eyes[0], blinks[0])
+        return probs, blinks
+
+class BlinkEstimatorPathPair(BlinkEstimatorStandalone):
+    def __init__(self, model_path, threshold, input_size, viz):
+        super(BlinkEstimatorPathPair, self).__init__(model_path, threshold, input_size, viz)
+        
+    def estimate(self, eye_paths):
+        right_eyes = [self.load_img(eye_paths[0])]
+        left_eyes = [self.load_img(eye_paths[1])]
+        probs, blinks = self.predict(right_eyes, left_eyes)
+
+        if self.viz:
+            #TODO: viz on eye images?
+            viz_img = self.overlay_prediction_over_img(right_eyes[0], blinks[0])
+        return probs, blinks
+
+class BlinkEstimatorFolder(BlinkEstimatorStandalone):
+
+    def estimate(self, folder_path):
+        right_eyes = []
+        left_eyes = []
+        for file_name in [f for f in listdir(folder_path) if 'right' not in f]:
+            right_eye_path = folder_path + '/' + file_name.replace('left', 'right')
+            left_eye_path = folder_path + '/' + file_name
+            right_eyes.append(self.load_img(right_eye_path))
+            left_eyes.append(self.load_img(left_eye_path))
+        probs, blinks = self.predict(right_eyes, left_eyes)
+        if self.viz:
+            #TODO: viz on eye images? cv2.imshow ?
+            viz_img = self.overlay_prediction_over_img(right_eyes[0], blinks[0])
+        return probs, blinks
+
+class BlinkEstimatorVideo(BlinkEstimatorStandalone):
+    # TODO
+    def estimate(self, video_path):
+        pass
+
+
+
+if __name__ == '__main__':
+    parser = argparse.ArgumentParser(description='Estimate blink from images, folder or even sequence.')
+    parser.add_argument('mode', type=str, default='image-pair', help='the input mode', choices=['image-pair', 'folder', 'video'])
+    parser.add_argument('--left', type=str, default=os.path.join(script_path, '../samples/'), nargs='?', help='Path to an image or a directory containing images')
```

Comment:
This is not very intuitive. The default for `--left` and `--right` seems to be the same?!?